opslevel/resource_opslevel_integration_google_cloud.go

```diff
@@ -0,0 +1,277 @@
+		Name:                  RequiredStringValue(googleCloudIntegration.Name),
+		OwnershipTagKeys:      StringSliceToSetValue(googleCloudIntegration.GoogleCloudIntegrationFragment.OwnershipTagKeys),
+		PrivateKey:            givenModel.PrivateKey,
+		TagsOverrideOwnership: types.BoolValue(googleCloudIntegration.GoogleCloudIntegrationFragment.TagsOverrideOwnership),
+	}
+
+	projects := make([]googleCloudProjectResourceModel, len(googleCloudIntegration.Projects))
+	for i, project := range googleCloudIntegration.Projects {
+		projects[i] = googleCloudProjectResourceModel{
+			ID:   RequiredStringValue(project.ID),
+			Name: RequiredStringValue(project.Name),
+			URL:  RequiredStringValue(project.URL),
+		}
+	}
+	projectsList, tmp := types.ListValueFrom(ctx, types.ObjectType{AttrTypes: googleCloudProjectAttrs()}, projects)
+	diags.Append(tmp...)
+	if diags.HasError() {
+		return integrationGoogleCloudResourceModel{}
+	}
+	resourceModel.Projects = projectsList
+
+	return resourceModel
+}
+
+func (r *integrationGoogleCloudResource) Metadata(ctx context.Context, req resource.MetadataRequest, resp *resource.MetadataResponse) {
+	resp.TypeName = req.ProviderTypeName + "_integration_google_cloud"
+}
+
+func (r *integrationGoogleCloudResource) Schema(ctx context.Context, req resource.SchemaRequest, resp *resource.SchemaResponse) {
+	resp.Schema = schema.Schema{
+		MarkdownDescription: "Google Cloud Integration resource",
+
+		Attributes: map[string]schema.Attribute{
+			"aliases": schema.ListAttribute{
+				Description: "All of the aliases attached to the integration.",
+				Computed:    true,
+				ElementType: types.StringType,
+			},
+			"client_email": schema.StringAttribute{
+				Description: "The service account email OpsLevel uses to access the Google Cloud account.",
+				Required:    true,
+				PlanModifiers: []planmodifier.String{
+					stringplanmodifier.RequiresReplace(),
+				},
+			},
+			"private_key": schema.StringAttribute{
+				Description: "The private key for the service account that OpsLevel uses to access the Google Cloud account.",
+				Required:    true,
+				Sensitive:   true,
+			},
+			"created_at": schema.StringAttribute{
+				Description: "The time this integration was created.",
+				Computed:    true,
+			},
+			"id": schema.StringAttribute{
+				Description: "The ID of the integration.",
+				Computed:    true,
+				PlanModifiers: []planmodifier.String{
+					stringplanmodifier.UseStateForUnknown(),
+				},
+			},
+			"installed_at": schema.StringAttribute{
+				Description: "The time that this integration was successfully installed, if null, this indicates the integration was not completely installed.",
+				Computed:    true,
+			},
+			"name": schema.StringAttribute{
+				Description: "The name of the integration.",
+				Required:    true,
+			},
+			"ownership_tag_keys": schema.SetAttribute{
+				ElementType: types.StringType,
+				Description: "An Array of tag keys used to associate ownership from an integration. Max 5",
```

Comment:
If this truly works like a `set` then we should not call it an `array` in the description.

Reply:
@rocktavious sets have been removed completely from this integration resource's code, schema and tests